src/WhyDoesThisFileExist.md

```diff
@@ -0,0 +1 @@
+hie-stack and stack produce different PWDs for template haskell. If this symlink exists, ExBudgetingDefaults works fine in both HIE and stack.
```

Comment:
I'd really prefer not to pollute the top-level with a mystery `src` dir. Could we use something like this? http://hackage.haskell.org/package/th-utilities-0.2.3.1/docs/TH-RelativePaths.html

Reply:
Doesn't help. I could do something different :-)